en/getting_started/ros.md

```diff
@@ -0,0 +1,12 @@
+# Generate Source Files for ROS
+
+If you want to add MAVlink messages or dialects while working with ROS, you need to:
+
+1. follow the [MAVROS source install instructions](https://github.com/mavlink/mavros/blob/master/mavros/README.md#source-installation) to install the mavlink-gbp-release which is the MAVlink library released for ROS.
+2. if you have previously installed the MAVlink package for ROS, uninstall it
+```
+sudo apt-get remove ros-${rosversion -d}-mavlink
+```
+where `*` should be replaced with the name of the ROS distribution you are using, e.g. `ros-kinetic-mavlink`.
```

Comment:
This line is not needed.

Reply:
Also uninstalling the previous version of `mavlink` for ROS is not required. You just need to source you workspace.